search-app/src/cmr/search/services/acl_service.clj

```diff
@@ -23,15 +23,36 @@
   [context acls concept]
   true)
 
+;; When plans solidify around acl checks for service concepts, we'll update this.
+(defmethod acls-match-concept? :service
+  [context acls concept]
+  true)
+
 (defmethod acls-match-concept? :default
   [context acls concept]
   false)
 
+;; XXX This code is a point of contention for adding new concepts: on both
```

Comment:
This is the second time this has bitten us. We need to incorporate this into our future plans for refactoring the code for adding new concepts to the CMR.